core/__tests__/screens/ProofDetails.test.tsx

```diff
@@ -0,0 +1,151 @@
+import { useProofById } from "@aries-framework/react-hooks";
+import mockRNCNetInfo from "@react-native-community/netinfo/jest/netinfo-mock";
+import { useNavigation } from "@react-navigation/core";
+import "@testing-library/jest-native/extend-expect";
+import { act, cleanup, fireEvent, render } from "@testing-library/react-native";
+import React from "react";
+
+import { testIdWithKey } from "../../App/utils/testable";
+import ProofDetails from "../../App/screens/ProofDetails";
+
+jest.mock("react-native/Libraries/EventEmitter/NativeEventEmitter");
+jest.mock("@react-native-community/netinfo", () => mockRNCNetInfo);
+jest.mock("react-native/Libraries/Animated/NativeAnimatedHelper");
+jest.mock("@react-navigation/core", () => {
+  return require("../../__mocks__/custom/@react-navigation/core");
+});
+jest.mock("@react-navigation/native", () => {
+  return require("../../__mocks__/custom/@react-navigation/native");
+});
+
+jest.useFakeTimers("legacy");
+jest.spyOn(global, "setTimeout");
+
+const requestPresentationMessage = new V1RequestPresentationMessage({
+  comment: "some comment",
+  requestPresentationAttachments: [
+    new Attachment({
+      id: INDY_PROOF_REQUEST_ATTACHMENT_ID,
+      mimeType: "application/json",
+      data: new AttachmentData({
+        json: {
+          name: "test proof request",
+          version: "1.0.0",
+          nonce: "1",
+          requestedAttributes: {
+            email: {
+              name: "email"
+            },
+            time: {
+              names: ["time"]
+            }
+          },
+          requestedPredicates: {}
+        }
+      })
+    })
+  ]
+});
+
+describe("displays a proof details screen", () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe("with a verified history proof record details", () => {
+    const testVerifiedProofRequest = new ProofExchangeRecord({
+      connectionId: "123",
+      threadId: requestPresentationMessage.id,
+      state: ProofState.RequestReceived,
+      protocolVersion: "V1",
+      isVerified: true
+    });
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+
+      // @ts-ignore
+      useProofById.mockReturnValue(testVerifiedProofRequest);
+    });
+
+    test("Renders correctly", async () => {
+      const tree = render(
+        <ProofDetails navigation={useNavigation()}
+                      route={{ params: { recordId: "", isHistory: true } } as any} />);
+      await act(async () => null)
+      expect(tree).toMatchSnapshot()
+    });
+
+    test("Renders history view correctly", async () => {
+      const { getByTestId } = render(
+        <ProofDetails navigation={useNavigation()}
+                      route={{ params: { recordId: "", isHistory: true } } as any} />);
+      const historyView = await getByTestId(testIdWithKey("HistoryView"));
+      expect(historyView).not.toBeNull();
```

Comment:
what is the difference between these two tests?

Reply:
I think one test should be with `isHistory: true` and another one with `isHistory: false`

Reply:
done